docs/DEV.md

```diff
@@ -167,7 +167,7 @@ The web extension needs to be packaged together with a legacy add-on in order to
 
 It is recommended to build necessary logic and user interface using in the context of the webextension and communicate with the legacy add-on code through messaging whenever privileged access is required.
 
-For more information, see [./about.md]
+For more information, see <https://github.com/mozilla/shield-studies-addon-utils/blob/master/docs/engineering.md>.
```

Comment:
This link doesn't work... I don't see an engineering.md in that repo.

Reply:
I think docs/engineering.md was added in [PR #113](https://github.com/mozilla/shield-studies-addon-utils/pull/113/files#diff-06447854e2fbc871cd227e16c82c1962) in that repo.